packages/meter/src/Meter.ts

```diff
@@ -45,9 +45,6 @@ export class Meter extends SizedMixin(ObserveSlotText(SpectrumElement, ''), {
     @property({ type: Number })
     public progress = 0;
 
-    @property({ type: Boolean, reflect: true, attribute: 'over-background' })
```

Comment:
Removing this amounts to a breaking change. We're still pre-1.0 so we track breaking changes as `feat(meter): ...` so that the commit will move the `0.x.0` value in our semvar.

Reply:
We can create a breaking change follow PR and revert not worry about this here maybe?